### DISCUSSION TITLE
Is there any methods to send signal to server for hot config reloading?

### DISCUSSION THREAD

**Discussion by brandousov** [bluenviron/mediamtx#367] [Questions]
Hi! We use server with our REST API for RTSP. When script make an updates into `rtsp-config.yml` the server often not reacting to file changes. How to fix this? Additional `touch` command in the script not help too. Only `systemctl restart mediaApi` helps, but it makes all open sessions broken for clients.

Thank you!

OS: Ubuntu server 20
Server: 0.15.5 (latest)

**Comment by aler9** [comment]
Hello, this will be solved when an HTTP API is added to the server. The other option (signals) is not portable and works on Linux only.

[a comment by an automated account is not shown]
